client.js

```diff
@@ -24,25 +28,41 @@ let client;
 const bigBucketOfData = {};
 const ring = new Set([]);
 let lastNode;
+const NULL_USER = { id: null };
 
-function fetch({ _: args }) {
-  if (!args[0]) {
+async function lookup({ _, ...rest }) {
+  if (!rest.id) {
     console.log("fetch required an ID");
     process.exit();
   }
-  id = parseInt(process.argv[3], 10);
-  client.fetch({ id }, (err, user) => {
-    if (err) {
-      console.log(err);
-    } else {
-      console.log(user);
-    }
-  });
+  console.log("Beginning client-side lookup: ", rest.id);
+  const lookedUpUser = await client.lookup({ id: rest.id });
+  console.log(lookedUpUser);
+  if (!lookedUpUser) {
+    console.log("Finished client-side lookup, found nothing");
+  } else {
+    console.log("Finished client-side lookup: ", lookedUpUser);
+  }
+}
+
+function remove({ _, ...rest }) {
+  if (!rest.id) {
+    console.log("remove required an ID");
+    process.exit();
+  }
+  console.log("Beginning client-side remove: ", rest.id);
+  try {
+    client.remove({ id: rest.id });
+    console.log("Remove successful, finishing");
```

Comment:
I would expect a "does not exist" gRPC code here if we try to remove something that doesn't exist. We can defer that until later, but you add a comment here to the effect that success does not necessarily mean the user was there in the first place.